src/util/sleep.js

```diff
@@ -0,0 +1,3 @@
+export async function sleep(ms) {
```

Comment:
is this being used?

Reply:
Not anymore. I sometimes use it when testing.

Reply:
ahh ok either way